### PR TITLE
Server-side check for reCaptcha user response

### PR DESCRIPTION
**Pull request by thesocialdev**
[#171548034](https://www.pivotaltracker.com/story/show/171548034)

### CODE REVIEW

File: packages/front/.env.example [hunk 1]
```diff
@@ -1 +1,2 @@
-API_URL=http://localhost:3000
\ No newline at end of file
+API_URL=http://localhost:3000
+RECAPTCHA_SITEKEY=6Lc2BtYUAAAAAOUBI-9r1sDJUIfG2nt6C43noOXh
```
Comment (wellcda) [inline packages/front/.env.example:2]: @thesocialdev let's start adding newline at EOF on the next PRs

File: packages/server/.gitignore [hunk 2]
```diff
@@ -21,3 +24,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+config.yaml
```
Comment (wellcda) [inline packages/server/.gitignore:28]: @thesocialdev let's start adding newline at EOF on the next PRs

Comment (thesocialdev) [reply packages/server/.gitignore]: Done.

File: packages/front/.env.example (hunk 1 above)
Comment (thesocialdev) [reply packages/front/.env.example]: Done.

**Pull request merged**